app/lib/screens/EditorScreen/Editor.dart

```diff
@@ -0,0 +1,320 @@
+import 'package:effektio/common/store/Colors.dart';
+import 'package:effektio/common/widget/TagItem.dart';
+import 'package:file_picker/file_picker.dart';
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
+import 'package:fluttertoast/fluttertoast.dart';
+import 'package:html_editor_enhanced/html_editor.dart';
+
+void main() => runApp(const HtmlEditorExampleApp());
+
+var arr = [
+  Colors.grey,
+  Colors.purple[300],
+  Colors.pink[100],
+  Colors.teal[400],
+  Colors.purple[700],
+  Colors.orange,
+  Colors.blue[300],
+  Colors.green,
+  Colors.yellow[100],
+  Colors.yellow,
+  Colors.pink,
+  Colors.red
+];
+
+List<String> _tagList = [];
+
+class HtmlEditorExampleApp extends StatelessWidget {
+  const HtmlEditorExampleApp({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      title: 'Editor',
+      theme: ThemeData.dark(),
+      darkTheme: ThemeData.dark(),
+      home: const HtmlEditorExample(title: 'Create FAQ'),
+    );
+  }
+}
+
+class HtmlEditorExample extends StatefulWidget {
+  const HtmlEditorExample({Key? key, required this.title}) : super(key: key);
+
+  final String title;
+
+  @override
+  _HtmlEditorExampleState createState() => _HtmlEditorExampleState();
+}
+
+class _HtmlEditorExampleState extends State<HtmlEditorExample> {
+  String result = '';
+  OverlayEntry? entry;
+  TextEditingController tagTitleController = TextEditingController();
+
+  final HtmlEditorController controller = HtmlEditorController();
+
+  @override
+  void initState() {
+    super.initState();
+    _tagList.clear();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return GestureDetector(
+      onTap: () {
+        if (!kIsWeb) {
+          controller.clearFocus();
+        }
+      },
+      child: Scaffold(
+        appBar: AppBar(
+          backgroundColor: AppColors.backgroundColor,
+          leading: TextButton(
+            onPressed: () {
+              Navigator.pop(context);
```

Comment:
closing the editor, crashes for me:

![Screenshot_1652475584](https://user-images.githubusercontent.com/40496/168389193-fb80eeee-ef5f-4bfa-8cc0-0f7d755db40e.png)

Reply:
Also while opening an editor, sometimes the application gets to not responding state or either immediate crash. But it might be related to #94 as I get the same logs.

Reply:
Also while in faq screen, when application gets in not responding state. I get the following log:
```
I/libc    (21990): Requested dump for tid 21990 (rg.effektio.app)

```